### PR TITLE
Pelumi/update/syncing UI library

### PR DESCRIPTION
**Pull request by Oluwadaminiola**
synced core with current ui library

### CODE REVIEW

File: web/ui/dashboard/src/styles.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // Material design themeing
 @use '@angular/material' as mat;
+@import 'https://frain-dev.github.io/ui-library/scss/main.scss';
```
Comment (horlah) [inline web/ui/dashboard/src/styles.scss:3]: Please remove

File: web/ui/dashboard/src/styles.scss [hunk 2]
```diff
@@ -43,28 +44,12 @@ $convoy-ng-theme: mat.define-light-theme(
...
-@import './scss/reset';
-@import './scss/button';
-@import './scss/card';
-@import './scss/form';
-@import './scss/table';
-@import './scss/tabs';
+// @import './scss/button';
+// @import './scss/card';
+// @import './scss/form';
+// @import './scss/table';
```
Comment (horlah) [inline web/ui/dashboard/src/styles.scss:50]: Do you still need these?

File: web/ui/dashboard/src/styles.scss (hunk 1 above)
Comment (horlah) [reply web/ui/dashboard/src/styles.scss:3]: Still pending @Oluwadaminiola

File: web/ui/dashboard/src/styles.scss (hunk 2 above)
Comment (horlah) [reply web/ui/dashboard/src/styles.scss:50]: Still pending @Oluwadaminiola

**Review by horlah — CHANGES_REQUESTED** [review]

**Comment by horlah** [comment]
Please update this branch with changes in `main` @Oluwadaminiola

**Comment by horlah** [comment]
Screenshot 2022-03-03 at 3 19 33 PM

You missed this dropdown list style. @Oluwadaminiola

**Comment by Oluwadaminiola** [comment]
> 

on it

**Comment by horlah** [comment]
@Oluwadaminiola Please don't publish the update yet. Let's merge this update first, there is another PR I want to add to the next publication.
Also, this update is a blocker for me to complete some things here.

**Comment by horlah** [comment]
Screenshot 2022-03-03 at 4 03 32 PM

The space between the list item is too small, see the one below.

Screenshot 2022-03-03 at 4 04 37 PM

@Oluwadaminiola

**Comment by horlah** [comment]
Screenshot 2022-03-03 at 4 11 34 PM

@Oluwadaminiola Did you review this section?

**Comment by horlah** [comment]
Screenshot 2022-03-03 at 4 35 10 PM

There is no space on the sides of the highlighted item on the table, check the design or previous implementation. @Oluwadaminiola

**Pull request merged**